packages/form/test/RadioGroup.astro.test.js

```diff
@@ -87,4 +87,28 @@ describe('RadioGroup.astro test', () => {
 		// assert
 		expect(actualResult).to.contain(expectedResult);
 	});
+
+	it('Should render readOnly fields if the flag is passed as true', async () => {
+		// arrange
+		const expectedOptions = 3;
+		const element = /disabled/g;
```

Comment:
After you added the `readonly` prop as suggested in my previous comment, you can change this to test for it instead of disabled. :)

Reply:
I keeped both tests, for disabled and readonly to keep testing all the scenários